src/RequestParser.php

```diff
@@ -156,6 +161,16 @@ private function parseSorter(?string $sortQuery): ?array
         return $sortStructs;
     }
 
+    private function parsePaginator($limitQuery, $offsetQuery){
+        if ($limitQuery xor $offsetQuery){
```

Comment:
Is it possible if we add some default value to the limit and offset ?
So we don't need to throw this exception.

Reply:
Default value of offset in query builder is 0. So we can use that.
As for the limit, I don't think it's appropriate to give it default value. Even query builder throws exception if the limit is empty

Reply:
But we can left out the limit, right ? Let the user get the whole data.
Another alternative is we can set the default value on config file.
WDYT ?

Reply:
Yes. So let me recap:
1. It's okay if the url doesn't contain both limit and offset. In this case, the query will proceed as usual (get the whole data)
2. If the limit is set and the offset isn't, we use 0 as default value.

Now we are left with the condition where the offset is set but the limit isn't. Looking at Laravel query builder, this will throw an exception. But we can also take another approach like ignoring the offset. Which one do you prefer? Throw an Exception or Ignore ?

Reply:
I prefer ignoring the offset.
Since offset without limit is useless.

Reply:
okay